### PR TITLE
Update availability zone with Europe

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- Merge to `rhacs-docs`
- Cherry pick to `rhacs-docs-4.0`

[Issue](https://issues.redhat.com/browse/ROX-15869)

Link to docs preview:

- OCP: https://57970--docspreview.netlify.app/openshift-acs/latest/installing/installing_cloud_ocp/cloud-create-instance-ocp.html
- non-OCP: https://57970--docspreview.netlify.app/openshift-acs/latest/installing/installing_cloud_other/cloud-create-instance-other.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cloud-create-instance-steps.adoc [hunk 1]
```diff
@@ -20,9 +20,11 @@ To create an *ACS instance*:
 . From the navigation menu, select *Advanced Cluster Security* -> *ACS Instances*.
 . Select *Create ACS instance* and enter information into the displayed fields or select the appropriate option from the drop-down list:
 * *Name*: Enter the name of your *ACS instance*. An *ACS instance* contains the {product-title-short} Central component, also referred to as "Central," which includes the {product-title-managed-short} management interface and services that are configured and managed by Red Hat. You manage your secured clusters that communicate with Central. You can connect multiple secured clusters to one instance.
-* *Cloud provider*: The cloud provider where Central is located. Select *AWS*. 
-* *Cloud region*: The region for your cloud provider where Central is located. Select *AWS*.
-* *Availability zones*: Use the default value.
+* *Cloud provider*: The cloud provider where Central is located. Select *AWS*.
+* *Cloud region*: The region for your cloud provider where Central is located. Select one of the following:
+** US-East, N. Virginia
+** Europe, Frankfurt
```
Comment (ebensh) [inline modules/cloud-create-instance-steps.adoc:26]: This has been updated to Dublin already (eu-west-1), the new text is "Europe, Ireland"

image

File: modules/cloud-create-instance-steps.adoc [hunk 2]
```diff
@@ -20,9 +20,11 @@ To create an *ACS instance*:
 . From the navigation menu, select *Advanced Cluster Security* -> *ACS Instances*.
 . Select *Create ACS instance* and enter information into the displayed fields or select the appropriate option from the drop-down list:
 * *Name*: Enter the name of your *ACS instance*. An *ACS instance* contains the {product-title-short} Central component, also referred to as "Central," which includes the {product-title-managed-short} management interface and services that are configured and managed by Red Hat. You manage your secured clusters that communicate with Central. You can connect multiple secured clusters to one instance.
-* *Cloud provider*: The cloud provider where Central is located. Select *AWS*. 
-* *Cloud region*: The region for your cloud provider where Central is located. Select *AWS*.
-* *Availability zones*: Use the default value.
+* *Cloud provider*: The cloud provider where Central is located. Select *AWS*.
+* *Cloud region*: The region for your cloud provider where Central is located. Select one of the following:
```
Comment (lahinson) [inline modules/cloud-create-instance-steps.adoc:24]:
```suggestion
* *Cloud region*: The region for your cloud provider where Central is located. Select one of the following regions:
```

Comment (lahinson) [reply modules/cloud-create-instance-steps.adoc:24]: From IBM Style:

> **following** _adjective_, _noun_
To introduce a list, procedure, or subsequent items, always use _following_ as an adjective before a noun. For example, do not write “Complete the following” or ”The functions include the following”. Instead, write “Complete the following steps” or “The adapter has the following functions”.

**Review by lahinson — COMMENTED** [review]
Just one tiny nit. Otherwise, LGTM!

**Comment by ousleyp** [comment]
This LGTM! Merging shortly.

**Pull request merged**